src/page/page2.html

```diff
@@ -94,23 +102,45 @@ <h1 class="title">Set your MultiMC path</h1>
               Continue
             </button>
             <a
-              class="forgot-password"
+              class="forgot-password open"
               href="https://multimc.org/"
-              target="_blank"
+              id="learn-more"
+              onclick="update_href(this.id)"
             >
               Learn More
             </a>
-            <a class="forgot-password" href="/finished?install=true">
+            <a
+              class="forgot-password open"
+              href="https://github.com/TouwaStar/Galaxy_Plugin_Minecraft/blob/fog_release/INSTALL_MULTIMC.md"
+              id="install-multimc"
+              onclick="update_href(this.id)"
+            >
               Install MultiMC
             </a>
+            <a
+              class="forgot-password"
+              href="/finished?install_mc=true"
+              id="install-mc"
+              onclick="update_href(this.id)"
+            >
+              Install Minecraft
+            </a>
```

Comment:
There is already a lot of info that overwhelm user. I prefer separate page for (Install Minecraft prompt & warning from above) that would be open if:
- user types any MultiMC path
- Vanila Minecraft is not installed (optional - nice to have)

Reply:
not obligatory

Reply:
Yeah, I see the issue with an overwhelming amount of info. It's definitely something that can be made better but unfortunately, I won't be able to implement it right now but instead, I'll add it to the todo.